pkgs/top-level/all-packages.nix

```diff
@@ -32563,7 +32565,9 @@ with pkgs;
 
   linkchecker = callPackage ../tools/networking/linkchecker { };
 
-  tomb = callPackage ../os-specific/linux/tomb {};
+  tomb = callPackage ../os-specific/linux/tomb {
+    pinentry = pinentry-curses;
```

Comment:
Do you know if using a single pinentry flavor is actually fine here (AFAIU these packages got the full package before).

Reply:
Yup, this not what we want. Have to revisit this and use the UI toolkit the respective software is using anyway.

Reply:
Now I remember. Problem is that neither tomb nor trezor depend on a specific UI toolkit. What should we default to?